Src/FluentAssertions/Types/MethodInfoSelector.cs

```diff
@@ -133,6 +133,24 @@ public MethodInfoSelector ThatAreNotDecoratedWithOrInherit<TAttribute>()
             return this;
         }
 
+        /// <summary>
+        /// Only return methods that are async. 
+        /// </summary>
+        public MethodInfoSelector ThatAreAsync()
+        {
+            selectedMethods = selectedMethods.Where(method => method.IsAsync());
+            return this;
+        }
+
+        /// <summary>
+        /// Only return methods that are not async. 
+        /// </summary>
+        public MethodInfoSelector ThatAreNotAsync()
```

Comment:
How about `ThatAreSync`?
Maybe you could also add a description for your motivation for these changes.

Reply:
`Sync` is the keyword that every developer knows. It also fits into the current naming pattern. I would in the future suggest the same when it comes to e.g. static. Then we would have `ThatAreNotStatic` instead of `ThatAreInstanceMethods`.